client/src/app/components/Layout/Layout.jsx

```diff
@@ -211,9 +197,11 @@ Layout.propTypes = {
   location: PropTypes.shape({
     pathname: PropTypes.string.isRequired
   }),
+  settings: PropTypes.shape({
+    viewType: PropTypes.string
```

Comment:
The type would be `oneOf([ViewType.LIST, ViewType.TILES]).isRequired`.

Reply:
I had to remove settings from prop validation because settings are loaded from localStorage directly in Layout component and they aren't passed as a prop from parrent component now.